### DISCUSSION TITLE
Toki Pona

### DISCUSSION THREAD

**Discussion by m-rey** [8VIM/8VIM#256] [Language Layout]
I've been learning [Toki Pona](https://wikipedia.org/wiki/Toki_Pona), a very simple conlang, and would like to use the 8VIM keyboard to write Toki Ppona, using an optimized layout for it.

**Describe the solution you'd like**
I'd like to have and be able to have a Toki Pona optimized layout for 8VIM. Preferably directly selectable from within the app, but importing the xml layout file manually would also work. SwiftKey is one of the few keyboards offering support for toki pona. Adding support to 8VIM would make this project stand out even more.

 I'm open to submit a PR for this if I get some guidance on how to best generate the layout.
I'll be scraping Toki Pona text to generate a dataset to calculate word/char frequencies etc. I assume this will be needed to generate an optimized layout.

What steps would be needed to make this feature request happen? Are there any tools already available that could help?

**Comment by flide** [comment]
Can you draw and upload, what the layout for Toki Pona would look like?

**Comment by qupufu** [comment]
Was thinking about this and 8vim has the ability to have 32 characters, so if you don't have punctuation, each sector could have the same letter on both sides. Haven't played around with it much at all but might work or might not

This thread actually introduced me to toki pona, so thanks :P
